src/t2iapi/metric/types.proto

```diff
@@ -65,3 +65,15 @@ enum MetricStatus {
     METRIC_STATUS_DE_INITIALIZED_AND_NOT_PERFORMING_OR_APPLYING = 4;
     METRIC_STATUS_FAILED = 5;
 }
+
+/*
+Represents ActivationState from the IEEE Std 11073-10207-2017.
+ */
+enum ActivationState {
```

Comment:
this is a duplicate for ComponentActivation, it seems to be not needed imo

Reply:
done